share/spack/gitlab/cloud_pipelines/stacks/build_systems/spack.yaml

```diff
@@ -36,12 +36,57 @@ spack:
       - spack env activate --without-view .
       - spack config add "config:install_tree:projections:${SPACK_JOB_SPEC_PKG_NAME}:'morepadding/{architecture}/{compiler.name}-{compiler.version}/{name}-{version}-{hash}'"
       - spack -d ci rebuild
+    image:
+      name: "ghcr.io/spack/e4s-ubuntu-18.04:v2021-10-18"
+      entrypoint: [ "" ]
+    tags:
+      - glr-testing
 
     mappings:
-      - match: [ 'os=ubuntu18.04' ]
+      - match:
+          - cmake
```

Comment:
I'm not sure `cmake` needs the huge, I think so far we've let it land in the catch-all, which is usually mapped to the smallest instance type.